src/coreclr/gc/vxsort/introsort.cpp

```diff
@@ -0,0 +1,108 @@
+
+#include "common.h"
+#include "introsort.h"
+
+
+ void introsort::sort (uint8_t** begin, uint8_t** end, int ignored)
+    {
```

Comment:
nit: formatting / extra spaces
```suggestion
void introsort::sort (uint8_t** begin, uint8_t** end, int ignored)
{
```